doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -54,7 +54,7 @@ High Availability for mgmt-gateway service
 
 In addition to providing high availability for the underlying backend services, the mgmt-gateway
 service itself can be configured for high availability, ensuring that the system remains resilient
-even if certain core components for the service fail.
+even if certain core components for the service fail, including the `mgmt-gateway` itself.
```

Comment:
Markdown rendering varies, but with Ceph docs I usually use double backticks.  Single backticks *might* be rendered as italics.  Let's see how GH renders, which might be different than Sphinx.

`single`
``double``

GH renders both of those the same, but Sphinx might.  I suggest just using double as best practice.

Reply:
I shared these links in the orch standup today, but I'll share them again here in case someone stumbles across this in the future:

https://stackoverflow.com/a/57302493
https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup

IOW, if you are writing reStructuredText, use double backticks for code, command lines, etc. Do not use (unadorned) single backticks unless you know what you are doing with them (making hyperlinks, references, etc). You should not use them just to make text italic.

Reply:
thanks @phlogistonjohn